app/views/students/update.html

```diff
@@ -0,0 +1,145 @@
+<div ng-controller='StudentsUpdateCtrl as controller' class='students-update-container'>
```

Comment:
Once I've finished with the view of #64, we can check it is possible to use that page header style instead of this one. The rest of this style I think it would be OK.